mutt/address.c

```diff
@@ -998,11 +1008,14 @@ const char *mutt_addr_for_display(struct Address *a)
     return a->mailbox;
 
   local_mailbox = mutt_idna_intl_to_local(user, domain, MI_MAY_BE_IRREVERSIBLE);
+
   if (!local_mailbox)
     return a->mailbox;
```

Comment:
You're leaking `user` and `domain` here. Please free them as soon as you're done using them.